Docs/source/install/hpc/crusher.rst

```diff
@@ -144,6 +212,44 @@ Known System Issues
 
    January, 2023 (OLCFDEV-1284, AMD Ticket: ORNLA-130):
    We discovered a regression in AMD ROCm, leading to 2x slower current deposition (and other slowdowns) in ROCm 5.3 and 5.4.
-   Reported to AMD and fixed for the 5.5 release of ROCm.
 
-   Upgrade ROCm or stay with the ROCm 5.2 module to avoid.
+   June, 2023:
```

Comment:
Shall we just cross-link to :ref:`known Frontier issues <known-crusher-issues>` since they are so similar? :)